websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,168 @@
+			"/shows/collected": { state: "The most collected shows" },
+			"/movies/trending": { state: "Trending movies" },
+			"/movies/popular": { state: "Popular movies" },
+			"/movies/recommended": { state: "Recommended movies" },
+			"/movies/anticipated": { state: "Anticipated movies" },
+			"/movies/watched": { state: "The most watched movies" },
+			"/movies/collected": { state: "The most collected movies" },
+			"/movies/boxoffice": { state: "Top 10 grossing movies" },
+			"/calendars/my": { details: "Browsing my calendar" },
+			"/calendars/my/shows": {
+				details: "Browsing their calendar",
+				state: "Shows",
+			},
+			"/calendars/my/premieres": {
+				details: "Browsing their calendar",
+				state: "Premieres",
+			},
+			"/calendars/my/new-shows": {
+				details: "Browsing their calendar",
+				state: "New shows",
+			},
+			"/calendars/my/movies": {
+				details: "Browsing their calendar",
+				state: "Movies",
+			},
+			"/calendars/my/dvd": {
+				details: "Browsing their calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/calendars/shows": {
+				details: "Browsing calendar",
+				state: "Shows",
+			},
+			"/calendars/premieres": {
+				details: "Browsing calendar",
+				state: "Premieres",
+			},
+			"/calendars/new-shows": {
+				details: "Browsing calendar",
+				state: "New shows",
+			},
+			"/calendars/movies": {
+				details: "Browsing calendar",
+				state: "Movies",
+			},
+			"/calendars/dvd": {
+				details: "Browsing calendar",
+				state: "DVD & Blu-ray",
+			},
+			"/discover": { details: "Finding new shows/movies" },
+			"/apps": { details: "Browsing apps" },
+			"/vip": { details: "Browsing VIP features" },
+			"/settings": { details: "Browsing settings" },
+			"/auth/signin": { details: "Registering on Trakt" },
+			"/auth/login": { details: "Logging in to Trakt" },
+			"/widgets": { details: "Browsing widgets" },
+		};
+
+	for (const [path, data] of Object.entries(pages))
+		if (pathname.includes(path)) presenceData = { ...presenceData, ...data };
+
+	if (pathname.includes("/users")) {
+		const username = document.querySelector(
+			"#avatar-wrapper > .emojis-supported > a"
+		).textContent;
+		presenceData.buttons = [{ label: "Check profile", url: href }];
+		if (pathname.includes("/history"))
+			presenceData.details = `Browsing ${username}'s history`;
+		else if (pathname.includes("/progress"))
+			presenceData.details = `Browsing ${username}'s progress`;
+		else if (pathname.includes("/collection"))
+			presenceData.details = `Browsing ${username}'s collection`;
+		else if (pathname.includes("/ratings"))
+			presenceData.details = `Browsing ${username}'s ratings`;
+		else if (pathname.includes("/lists"))
+			presenceData.details = `Browsing ${username}'s lists`;
+		else if (pathname.includes("/comments"))
+			presenceData.details = `Browsing ${username}'s comments`;
+		else if (
+			pathname.includes("/network") ||
+			pathname.includes("/network/friends")
+		)
```

Comment:
this OR condition is redundant. If it includes /network/friends it will also include /network.
```suggestion
		else if (pathname.includes("/network"))
```